calin/nz/_posts/2022-07-24-the-move.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: The Move
+author: Călin Cașcaval
+---
+
+Moving house across continents is not an easy feat: although we took the opportunity
+to somewhat slim down when we moved houses in Oct. 2020, it was too easy to just
+"take it" when we mved one mile across town! This time however, we agreed that it
+makes sense to haul only the minimum number of things we want to keep. We'll be
+moving withing New Zealand as well until we find our place, and really, do we need
+to keep the paper credit card statements from the '90s?
+
+We contacted a few companies for moving and their first question was "what inventory
+are you moving?". So we sat down with a spreadsheed and sorted through our _stuff_:
+things we move, things we keep, and things we let go. For all the papers, we went
+full digital, so both our scanners did extra duty for a few months! Same thing with
+our collection of photos and films. As for letting go of other things ... not so
+easy. Anca has quite a number of stories to tell from talking to charities and other
+organizations that are geared toward helping the homeless but take only new stuff.
+What finally worked was a combination of charities, neighborhood groups such as [Buy
+Nothing](https://www.facebook.com/BuyNothingProject/){:target="_blank"} and [Free Cycle](https://www.freecycle.org/town/PaloAltoCA){:target="_blank"}, and
+Facebook marketplace -- turns out that for somethings you just have to put a price
+on in order to get attention! And so we kept unloading before the movers came, and
+then they came and loaded the stuff we said we'll take, and we were still left with
+half a house full of things! In the end, it was Anca's friends from the garden club
+who stepped in and helped us clean up everything else.
+
+It has been our plan all along to fly out from Seattle and spend some time with him
```

Comment:
him - who's him? (you haven't introduced me yet, I think!)

Reply:
I had introduced you in a previous version. Thanks!